tests/prepare/repository/test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
```

Comment:
IMO test should check that repo doesn't exist before and also clean the repo after it runs.
Also since this require root permission please tag the test with `as_root` tag so 'tests/full' can filter out properly this test (as it runs as a user)